app/components/ProductItem.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import {Link} from 'react-router'
+import removeProduct from './dispatchers/removeProduct'
+
+
+// we don't have api routes???
+
+
+
+class ProductItem extends React.Component {
+  constructor(props){
+    super(props)
+    this.removeProduct = this.removeProduct.bind(this)
+  }
+
+
+render(){
+  const {product} = props.product
```

Comment:
this.props.product -- also, not sure you need to declare it within curly braces if just pulling as a const.